docs/modules/ROOT/pages/getting-started/querying.adoc

```diff
@@ -1,48 +1,43 @@
 = Querying
 
-This tutorial shows how to create a simple Cypher query to match and return Movies using the Cypher Builder:
+This tutorial shows how to create a simple Cypher query to `MATCH` and `RETURN` Movies using the Cypher Builder:
 
 ```cypher
 MATCH(m:Movie)
 RETURN m
 ```
 
-== Matching nodes
-
-First, you need to define a node variable. You can do it by creating a new instance of `Cypher.Node`:
+== Instructions
```

Comment:
I see the other guides (filters and projection, relationships and advance querying) are not using a section named `Instructions` but a section describing what is going to be done. Is it a problem to follow the same pattern here for consistency?

Reply:
Good catch, it should actually be considered a guide like the Installation one: no need for further explanations and contextualization, just a list of what you should do. I will change that.